kia-lib/src/main/kotlin/com/faforever/ice/telemetry/TelemetryClient.kt

```diff
@@ -23,6 +24,31 @@ class TelemetryClient(
     private val websocketClient: WebSocketClient
     private var connectingFuture: CompletableFuture<Void>
 
+    inner class TelemetryWebsocketClient(serverUri: URI): WebSocketClient(serverUri) {
```

Comment:
This inner class just moved from below where it was anonymous so that it can be mocked in tests.  Maybe there's a better way but this is what I could get to work.

Reply:
That's fine, testability first.